src/librustdoc/passes/collect_intra_doc_links.rs

```diff
@@ -135,8 +135,16 @@ enum ResolutionFailure<'a> {
     ///
     /// `Namespace` is the namespace specified with a disambiguator
     /// (as opposed to the actual namespace of the `Res`).
```

Comment:
I think the comment is unnecessary now that the field has its own docs.

Reply:
Oops, yes: forgot to remove it.

Reply:
Can you please remove it now?

Reply:
Removed.